src/ckeditor.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* global CKEDITOR */
+
+import { getEditorNamespace } from './helpers.js';
+
+export default {
+	name: 'ckeditor',
+
+	render( createElement ) {
+		return createElement( 'div', {}, [
+			createElement( this.tagName )
+		] );
+	},
+
+	props: {
+		value: {
+			type: String,
+			default: ''
+		},
+		type: {
+			type: String,
+			default: 'classic',
+			validator: type => type in { classic: 1, inline: 1 }
+		},
+		editorUrl: {
+			type: String,
+			default: 'https://cdn.ckeditor.com/4.11.4/standard/ckeditor.js'
```

Comment:
Existing `CKEDITOR` namespace takes precedence over default, or even passed url. With default value the component will work out of the box, also this is consistent with both React and Angular integrations.

Reply:
I see how it's working. I don't understand why we need multiple ways to include CKEditor4 instance. Such hardcoded URL seems redundant and requires maintaining.

Reply:
I don't have strong opinion here. But maybe it's worth to have bigger discussion here. If we decide to drop hardcoding `editorUrl`, then we could do the same in other frameworks, they are still in `beta` status, so it's not so unexpected to have breaking changes.

Reply:
I would leave URL intact, as it allows lazy loading CKE just before creating editor's instance. Global `CKEditor` must be already bundled/loaded when someone attempts to create editor.

TBH I'm thinking about resolving this issue on entirely different level – introduce `latest` "version" to our CDN:

```javascript
'https://cdn.ckeditor.com/latest/standard/ckeditor.js'
```

OTOH we have already automated it in React, so copying and modifying script for Vue shouldn't be much work.

Reply:
Using "latest" in the URL and a CDN is kinda incompatible. You'd have to set a very very short caching level (and CDN's are rather not for that) in order to minimize the negative consequences of it. Suppose we set `max-age=3600, must-revalidate`. If we change the CKEditor library under the same URL, then there is a non-zero chance, that within the next hour a user who already loaded CKEditor 4 (`ckeditor.js`),will face issues caused by additional external resources were not loaded. Imagine e.g. a scenario where CKEditor ver. N tries to load e.g. a dialog window or a translation coming from version N+1. In some situations may lead to a broken editor.

Reply:
Well, the issue will actually occur always, regardless if the CDN is used or not, as long as you are not using a versioned folder.

Reply:
The cache of a CDN will eventually cause additional issues if someone creates a plugin which is using only the newest version of a plugin and count on people having the latest version of CKEditor served from a CDN. While some of them may have still cached the older version. Even cache invalidation on edge locations of a CDN does not help (in reality).

Reply:
I would be for keeping the `editorUrl` as it makes using integration easier. You just install one package and are ready to go. And it assures the user that integration works with the given editor version (which is provided by default).

Without default one it requires additional checking which CKEditor version you should/could use with the integration (well, ATM all version from like few years back should work fine, but user doesn't have to know or want to think about that). You can change version manually, but then if something breaks it's obvious that it's because of editor version change.